tests/test_install.py

```diff
@@ -448,5 +448,18 @@ def test_misc(self):
         self.assertEqual(duplicates_to_remove(li, [d1, d2]), [])
 
 
+def test_standalone_import():
+    import sys
+    import conda.install
+    tmp_dir = tempfile.mkdtemp()
+    fname = conda.install.__file__.rstrip('co')
+    shutil.copyfile(fname, join(tmp_dir, basename(fname)))
+    sys.path = [tmp_dir] + sys.path
+    try:
```

Comment:
Just prepending the temp dir to `sys.path` is not enough, because all of `conda` can still be imported.  So this wouldn't catch the unallowed imports.  I think we need so *limit* to `tmp_dir` and the standard library.